services/userService.js

```diff
@@ -0,0 +1,27 @@
+const userDao = require('../models/userDao');
+const { baseError } = require('../utils/error');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+
+const getUserByAccount = async (account) => {
+  return await userDao.getUserByAccount(account);
+};
+
+const signIn = async (account, password) => {
+  const user = await userDao.getUserByAccount(account);
+  if (!user) {
+    throw new baseError('USER_NOT_VALID');
+  }
+
+  const checkHash = await bcrypt.compare(password, user.password);
+
+  if (!checkHash) {
+    throw new baseError('PASSWORD_NOT_MATCH', 401);
```

Comment:
계정이 존재 하지 않을 떄와 비밀번호가 달랐을 때 에러 메시지를 통일하는 것이 좋습니다.
```suggestion
    throw new baseError('USER_NOT_VALID', 401);
```